src/endian.rs

```diff
@@ -25,21 +25,10 @@ pub trait FromByteArray<T> {
 
 macro_rules! define_endian {
     ($endian:ident) => {
+        #[derive(Clone, Copy)]
```

Comment:
Ditto. I guess we'll need a coverage test for this, which wasn't being tested either way. (It's not tested because it's also `Copy` and the `Copy` implementation is used instead of `Clone`, which we don't need, except that `Copy` forces us to implement it.)